Source/EventFlow/Extensions/SpecificationExtensions.cs

```diff
@@ -44,7 +45,21 @@ public static class SpecificationExtensions
             if (whyIsNotStatisfiedBy.Any())
             {
                 throw DomainError.With(
-                    $"'{specification.GetType().PrettyPrint()}' is not satisfied becase of {string.Join(" and ", whyIsNotStatisfiedBy)}");
+                    $"'{specification.GetType().PrettyPrint()}' is not satisfied because of {string.Join(" and ", whyIsNotStatisfiedBy)}");
+            }
+        }
+
+        public static void ThrowDomainErrorIfNotSatisfied<T>(
```

Comment:
Please merely invoke the new from the obsoleted version, no need for the code duplication

Reply:
Done